addOns/llm/src/main/java/org/zaproxy/addon/llm/ExtensionLlm.java

```diff
@@ -34,14 +36,14 @@
  * An extension for ZAP that enables researchers to leverage Large Language Models (LLMs) to augment
  * the functionalities of ZAP.
  */
-public class ExtensionLlm extends ExtensionAdaptor {
+public class ExtensionLlm extends ExtensionAdaptor implements OptionsChangedListener {
```

Comment:
This is not need, the options instance is always the same.

Reply:
Okay.